frontend/src/features/admin-form/responses/IndividualResponsePage/PaymentSection.tsx

```diff
@@ -101,7 +101,7 @@ const PayoutDataHeader = ({
         Depending on payment method, payouts happen 1 - 3 working days after a respondent makes payment."
       >
         <Flex justify="center" align="center">
-          <Icon as={BiInfoCircle} fontSize="1.5rem" ml="0.5rem" />
+          <Icon as={BiInfoCircle} fontSize="1.25rem" ml="0.5rem" />
```

Comment:
Oh FRM-1315 stated 16px right? was that too small?

Reply:
I went to check on figma, its `20px`, thus I used that instead. `16px` looked too small.

@staceytan1998 I'll need your help here.